cro-clib/src/fee.rs

```diff
@@ -0,0 +1,58 @@
+use crate::types::get_string;
+use crate::types::CroResult;
+use crate::types::{CroFee, CroFeePtr};
+pub use chain_core::init::network::Network;
+use chain_core::tx::fee::{LinearFee, Milli};
+use std::ffi::CString;
+use std::os::raw::c_char;
+use std::ptr;
+use std::str::FromStr;
+
+/// create fee
+#[no_mangle]
+/// # Safety
+pub unsafe extern "C" fn cro_create_fee(
```

Comment:
this is not creating a "fee", but "fee policy" (something that can be used to compute fees)

Reply:
so better adjust names and comments to make it clear

Reply:
Or all components share a config structure, which is inited with genesis file directly. So no need to copy-paste individuals parameters from genesis file.

Reply:
ok